pyvista/core/filters.py

```diff
@@ -794,9 +794,9 @@ def contour(dataset, isosurfaces=10, scalars=None, compute_normals=False,
         else:
             _, field = get_array(dataset, scalars, preference=preference, info=True)
         # NOTE: only point data is allowed? well cells works but seems buggy?
-        if field != pyvista.POINT_DATA_FIELD:
+        if field != pyvista.FieldAssociation.POINT:
```

Comment:
As `FiedAssociation` is directly imported, there's not need for the `pyvista.` prefix (didn't comment on further instances of this).